test/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/fieldwatch/fw001.java

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+package com.ibm.jvmti.tests.fieldwatch;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class fw001 {
+
+	public static AtomicInteger numReports = new AtomicInteger();
+	public static boolean success;
+	public static final int JNI_OK = 0;
```

Comment:
Pleaase don't do this - return a boolean instead (since you're only returning OK or ERR).

Reply:
I'm not sure I follow this comment. I'm not returning `JNI_OK` anywhere.

Reply:
You're essentially returning pass fail - there's no need for a magic C constant in the java code when the native can just return true or false.

Reply:
All uses of `JNI_OK` were removed as part of this commit: 650ce45e76c6d644d9f354df6172d2e1853ae96c